packages/block-library/src/paragraph/variations.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Path, SVG } from '@wordpress/primitives';
+import { paragraph } from '@wordpress/icons';
+
+const variations = [
+	{
+		name: 'paragraph',
+		title: __( 'Paragraph' ),
+		description: __(
+			'Start with the basic building block of all narrative.'
+		),
+		isDefault: true,
+		scope: [ 'block', 'inserter', 'transform' ],
```

Comment:
```suggestion
		scope: [ 'block', 'inserter' ],
```

Personally, I think it would be sufficient if stretched text could simply be inserted from the inserter, and that transformation is unnecessary. This transformation  UI in the sidebar is too prominent.

<img width="369" height="209" alt="Image" src="https://github.com/user-attachments/assets/81468b6d-6b14-47c0-99be-0144982d7c3b" />

The same applies to the Heading block.

cc @jasmussen

Reply:
Related to this line, we don't need `block` scope either.

Reply:
I'd agree with that, mostly to keep the inspector compact.

Reply:
At the same time, showing it here does increase discoverability. It's a tricky one.

Reply:
We could remove the transforms, that would simplify things and also, remove the need for default variations.
I think for now we can remove, and later in a future version if we want to increase visibility we can readd.

Reply:
Transforms were removed.